src/main/resources/logback-test.xml

```diff
@@ -0,0 +1,17 @@
+<configuration>
```

Comment:
In `main`, the logger config isn't needed as we don't run it as a standalone app. Moreover, it has some potential of interfering with the connect logging config in prod (though unlikely as Connect doesn't use Logback, but still).

Reply:
I meant to move it to src/test